Fix `simplified-invokes` TestGenerateProgram test

This test from pulumi/pulumi makes use of pulumi-std 0.1.0. I've tried updating the `get_plugins` and `update_plugin_docs` Makefile targets to install std 0.1.0, and updated the `get_schema.sh` file to download the schema from a custom url, similar to awx, e.g.:

```
std_url='https://raw.githubusercontent.com/pulumi/pulumi-std/v_VERSION_/sdk/schema.json'

"std@0.1.0@${std_url}"
```

But when I run the TestGenerateProgram test, `simplified-invokes` fails with:

```
panic: expected plugin loader to use cached schema path, but cache was missed for package std@1.0.0, is an entry in the makefile or setup for this package missing? [recovered]
panic: expected plugin loader to use cached schema path, but cache was missed for package std@1.0.0, is an entry in the makefile or setup for this package missing?
```

Not sure why it's trying to use 1.0.0 or how to address this, so I'm going to skip the test for now.

WIP change: https://github.com/pulumi/pulumi-yaml/compare/justin/simplified-invokes?expand=1

--- a/pkg/pulumiyaml/codegen/gen_program_test.go
+++ b/pkg/pulumiyaml/codegen/gen_program_test.go
@@ -194,8 +194,6 @@ func TestGenerateProgram(t *testing.T) {
 				l = append(l, tt)
 			case "traverse-union-repro":
 				// Reason: this example is known to be invalid
-			case "simplified-invokes":
-				// https://github.com/pulumi/pulumi-yaml/issues/438
 			case "regress-node-12507":
 				// https://github.com/pulumi/pulumi-yaml/issues/494
 			case "config-variables":

--- a/pkg/pulumiyaml/testing/test/testdata/simplified-invokes-pp/yaml/simplified-invokes.yaml
+++ b/pkg/pulumiyaml/testing/test/testdata/simplified-invokes-pp/yaml/simplified-invokes.yaml
@@ -1,0 +1,31 @@
+variables:
+  everyArg:
+    fn::invoke:
+      Function: std:AbsMultiArgs
+      Arguments:
+        a: 10
+        b: 20
+        c: 30
+  onlyRequiredArgs:
+    fn::invoke:
+      Function: std:AbsMultiArgs
+      Arguments:
+        a: 10
+  optionalArgs:
+    fn::invoke:
+      Function: std:AbsMultiArgs
+      Arguments:
+        a: 10
+        c: 30
+  nestedUse:
+    fn::invoke:
+      Function: std:AbsMultiArgs
+      Arguments:
+        a: ${everyArg}
+        b:
+          fn::invoke:
+            Function: std:AbsMultiArgs
+            Arguments:
+              a: 42
+outputs:
+  result: ${nestedUse}

